File: midi-test.cpp
#include <iostream>

#include "midi-tools.h"

int main() {
  std::cout << "The C minor scale: ";
  const Scale scale = getScale("A", 4, "aeolian");
  for (const int& value : scale) {
    std::cout << value << ' ';
  }
  std::cout << "\nThe C minor chord: ";
  const Chord chord = getChord("A", 4, "aeolian", 3);
  for (const int& value : chord) {
    std::cout << value << ' ';
  }
}

File: midi-test.js
import { getMidiNote, getScale, getChord } from './index.js';

const scale = getScale("A", "aeolian");
console.log('The A minor scale:', scale.join(' '));

const root = getMidiNote('A', 4);
let chord = getChord(root, 'A', "aeolian");
console.log('The A minor triad:', chord.join(' '));
chord = getChord(root, 'A', "aeolian", 'ninth');
console.log('The A minor ninth chord:', chord.join(' '));
